docs/Models.rst

```diff
@@ -233,6 +233,8 @@ These modules provide helper functions or are older versions of above modules.
       - Complex utility rate structure net revenue calculator OpenEI Version 4
     * - :doc:`modules/Utilityrate`
       - Complex utility rate structure net revenue calculator
+	* - :doc`modules/WaveFileReader`
+	  - Wave resource data file reader (CSV)
```

Comment:
Be careful with the spacing, it needs to match up with the others otherwise the table formatting is incorrect: https://github.com/NREL/pysam/blob/b72ae47e6e61caa86881744233717c8d4a02e133/docs/Models.rst